internal/test/framework/framework.go

```diff
@@ -242,6 +243,67 @@ func (f *TestFramework) getClusterVersion() error {
 	return nil
 }
 
+// GetK8sVersion gets the Kubernetes version in vx.x.x format
+func (f *TestFramework) GetK8sVersion() (string, error) {
+	versionInfo, err := f.OSConfigClient.Discovery().ServerVersion()
+	if err != nil {
+		return "", fmt.Errorf("error while retrieving k8s version : %s", err)
+	}
+
+	return versionInfo.GitVersion, nil
+
+}
+
+// Get KubeNodeUrl gets the url for kubernetes windows node binary.
+func (f *TestFramework) GetKubeNodeURL() (string, error) {
```

Comment:
Is this method going to be used by both wmcb and wsu? If not, this should go to `wsuFramework`.

Reply:
It is used by wmcb tests. Should I add it to `wmcbFramework` ?

Reply:
Yeah, wmcb sorry for the confusion.